### PR TITLE
Add description about pushing and popping view controller

### PR DESCRIPTION
**Pull request by dasdom**

### CODE REVIEW

File: UITests/UITests.swift [hunk 1]
```diff
@@ -149,6 +149,13 @@ class UITests: XCTestCase {
         app.buttons["More Info"].tap()
         XCTAssert(app.navigationBars["Volleyball?"].exists)
     }
+    
+    func testPopAViewController() {
+        app.buttons["More Info"].tap()
+        XCTAssert(app.navigationBars["Volleyball?"].exists)
+        app.navigationBars.elementBoundByIndex(0).childrenMatchingType(.Button).elementBoundByIndex(0).tap()
```
Comment (joemasilotti) [inline UITests/UITests.swift]: Can you change `elementBoundByIndex(0)` to `element`?

**Comment by joemasilotti** [comment]
Awesome, thanks for this! I just added one minor comment.

File: UITests/UITests.swift [hunk 2]
```diff
@@ -149,6 +149,13 @@ class UITests: XCTestCase {
         app.buttons["More Info"].tap()
         XCTAssert(app.navigationBars["Volleyball?"].exists)
     }
+    
+    func testPopAViewController() {
```
Comment (joemasilotti) [inline UITests/UITests.swift]: Oh, and please change "Pop" to "Popping" to keep the same tense as "Pushing".

File: README.md [hunk 3]
```diff
@@ -145,3 +145,19 @@ let start = firstCell.coordinateWithNormalizedOffset(CGVectorMake(0, 0))
...
+
+```swift
+app.buttons["More Info"].tap()
+XCTAssert(app.navigationBars["Volleyball?"].exists)
+```
+
+Pop a view controller by tapping the back button in the navigation bar and assert that the title in the navigation bar has changed.
+
+```swift
+app.navigationBars.elementBoundByIndex(0).childrenMatchingType(.Button).elementBoundByIndex(0).tap()
```
Comment (joemasilotti) [inline README.md]: Please apply the same `element` comment here, too.

File: UITests/UITests.swift (hunk 1 above)
Comment (dasdom) [inline UITests/UITests.swift]: If I try to do this:

```
app.navigationBars.element.childrenMatchingType(.Button).element.tap()
```

I get the error "Multiple matches found".

Could you describe further what you mean?

Comment (dasdom) [inline UITests/UITests.swift]: Never mind. Got it. I'll update my commit.

**Comment by dasdom** [comment]
Done. In addition I added a link in the table of contents.

**Comment by joemasilotti** [comment]
Looks great, merging in now. Thanks again!

**Pull request merged**